OpenGDPR_specification.md

```diff
@@ -193,13 +193,17 @@ erasure
 
   **REQUIRED** version string representing the supported version of the OpenGDPR API.
 
- `supported_identities`
+`supported_identities`
 
   **REQUIRED** array of "identity_type" and "identity_format" pairs.
 
 `supported_subject_request_types`
 
-   **REQUIRED** array of "subject_request_type" strings as defined in 6.2.
+  **REQUIRED** array of "subject_request_type" strings as defined in 6.2.
+
+`supported_extensions`
+
+  **REQUIRED** array of extension keys as defined in 7.1.2.
```

Comment:
- why is it required? If a processor does not support any extension - must they return an empty-list vs. null? 
- is it worth noting that this is with the intention that requests could be federated?

Reply:
- It keeps it in line with the others: all required. Someone could conceivably have 0 supported public identities (passing an empty array) and one or more supported extensions. What do you think?
- I don't think it needs to be called out since it could work by aggregating the discovery info.